gate_block/models.py

```diff
@@ -96,3 +111,27 @@ def __unicode__(self):
         return "gate %d submission by %s at %s" % (self.gateblock.id,
                                                    unicode(self.gate_user),
                                                    self.submitted)
+
+
+class SectionSubmission(models.Model):
+    section = models.ForeignKey(Section, related_name='section_submited')
+    user = models.ForeignKey(User, related_name='section_user')
+    submitted = models.DateTimeField(auto_now_add=True, editable=False)
+
+    @classmethod
+    def get_or_create(cls, section, user):
```

Comment:
Why are you defining your own `get_or_create()` on the model? It's not clear to me what this gets you over the one that comes for free on django's model manager.